apis/nucleus/src/viz.js

```diff
@@ -149,6 +150,44 @@ export default function viz({ model, halo, initialError, onDestroy = async () =>
       const propertyTree = await conversionConvertTo({ halo, model, cellRef, newType });
       return propertyTree;
     },
+    /**
+     * Toggles the view of the chart into an accessible table. The type (default sn-table) needs to be registered in order to use this functionality
+     *
+     * @since 4.6.0
```

Comment:
Should it maybe start as experimental?

Reply:
Sure, we could start by having it experimental...